### ISSUE TITLE
Test are not failing.

### ISSUE THREAD

**Issue by mahugnon** [moosetechnology/PowerBuilderParser#12]
PWBUserObjectGrammarTest>>testFunctionInvocation7
PWBUserObjectGrammarTest>>testFunctionInvocation8
PWBUserObjectGrammarTest>>testExpression1
PWBDataWindowGrammarTest>>testTable4
PWBDataWindowGrammarTest>>testType2
PWBDataWindowGrammarTest>>testType3

**Comment by mahugnon** [comment]
These tests are not passing because  of the rule for `m_-`

**Comment by Larcheveque** [comment]
> These tests are not passing because of the rule for `m_-`

no, the most of them is due to invocation with space before opening parenthesis for parameters
